docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -43,66 +42,16 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
 Your product team can only have a **permanent**, private repository in `bcgov-c` if it is a GitOps repository with ArgoCD manifests. 
 
 
-## Single sign-on is coming to the B.C. government’s GitHub organizations
+## Single sign-on 
 
-We enabled single sign-on (SSO) for GitHub. We’re starting with the [bcgov GitHub organization](https://github.com/bcgov). After we finish transitioning [bcgov](#bcgov), we’ll enable other OCIO-managed GitHub organizations. 
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
-
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to the bcgov GitHub organization, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled. 
 
 [Learn more about IDIRs](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/). 
- 
-[Learn more about the timelines and instructions](bc-government-organizations-in-github.md#the-changes). 
```

Comment:
we'll need some version of this, but lighter, for the migration to the bcgov-c GitHub organization. we should have dates for this by next week.

Reply:
I re-purposed the unrefined instructions and changes text the bcgov-c part for now, and once dates, etc. is finalized, I'll re-work the text.

Reply:
@jenreiher-bcgov  I re-purposed the unrefined instructions and changes text the bcgov-c part for now, and once dates, etc. is finalized, I'll re-work the text.